File: data/src/main/java/org/cryptomator/data/cloud/crypto/CryptoImplVaultFormat8.java
package org.cryptomator.data.cloud.crypto;

import android.content.Context;

import org.cryptomator.cryptolib.api.Cryptor;
import org.cryptomator.domain.CloudFolder;
import org.cryptomator.domain.repository.CloudContentRepository;
import org.cryptomator.util.Supplier;

public class CryptoImplVaultFormat8 extends CryptoImplVaultFormat7 {

	CryptoImplVaultFormat8(Context context, Supplier<Cryptor> cryptor, CloudContentRepository cloudContentRepository, CloudFolder storageLocation, DirIdCache dirIdCache, int shorteningThreshold) {
		super(context, cryptor, cloudContentRepository, storageLocation, dirIdCache, shorteningThreshold);
	}

}

File: data/src/main/java/org/cryptomator/data/cloud/crypto/CryptoImplVaultFormat8.kt
package org.cryptomator.data.cloud.crypto

import android.content.Context
import org.cryptomator.cryptolib.api.Cryptor
import org.cryptomator.domain.Cloud
import org.cryptomator.domain.CloudFile
import org.cryptomator.domain.CloudFolder
import org.cryptomator.domain.CloudNode
import org.cryptomator.domain.repository.CloudContentRepository
import java.util.function.Supplier

class CryptoImplVaultFormat8 internal constructor(
	context: Context,
	cryptor: Supplier<Cryptor>,
	cloudContentRepository: CloudContentRepository<Cloud, CloudNode, CloudFolder, CloudFile>,
	storageLocation: CloudFolder,
	dirIdCache: DirIdCache,
	shorteningThreshold: Int
) : CryptoImplVaultFormat7(
	context, cryptor, cloudContentRepository, storageLocation, dirIdCache, shorteningThreshold
)
